src/System.Net.Requests/ref/System.Net.Requests.cs

```diff
@@ -124,3 +124,59 @@ public abstract partial class WebResponse : System.IDisposable
         public abstract System.IO.Stream GetResponseStream();
     }
 }
+namespace System.Net.Cache
```

Comment:
This change broken consumption of the new packages with this change. You cannot add more APIs into the older netstandard1.3 version. See http://dotnet-ci.cloudapp.net/job/dotnet_corefx/job/master/job/windows_nt_debug_prtest/543/consoleFull

This needs to be bumped to netstandard1.7. 

cc @stephentoub @ericstj